index.html

```diff
@@ -918,6 +919,18 @@ <h2>RDF Serialization/Deserialization</h2>
     <a href="#serialize-rdf-as-json-ld-algorithm">serialize RDF as JSON-LD</a> as
     defined in this specification.</p>
 
+  <p class="changed">Unless specified using
+    <a data-link-for="JsonLdOptions">processingMode</a> API option,
+    the <a>processing mode</a> is set using the <code>@version</code> <a>entry</a>
+    in a local <a>context</a> and
+    affects the behavior of algorithms including <a>expansion</a> and <a>compaction</a>.
+    Once set, it is an error to attempt to change to a different processing mode,
+    and processors MUST generate,
```

Comment:
"you can't downgrade but you can upgrade?" sounds like a reasonable statement to me...

Reply:
I agree with @iherman. NB: if this is changed, this should be changed accordingly in the Framing doc.

Reply:
Agreed, you can upgrade; for 1.1, it would be an error to use 1.2, just as with 1.0, it's an error to use 1.1.

That said, it should never be necessary to detect things based on `@version` in the future, as we specifically check for things we expect in context processing, and reject anything else. We didn't do this properly in 1.0.